src/tests/integration/proof/test_refute_node.py

```diff
@@ -195,3 +190,60 @@ def test_apr_proof_refute_node(
 
         # Then
         assert prover.proof.status == expected_status
+
+    def test_apr_proof_read_node_refutations(
+        self,
+        kprove: KProve,
+        proof_dir: Path,
+        kcfg_explore: KCFGExplore,
+    ) -> None:
+        # Given
+        spec_file = K_FILES / 'refute-node-spec.k'
+        spec_module = 'REFUTE-NODE-SPEC'
+        claim_id = 'split-int-succeed'
+
+        prover = self.build_prover(kprove, proof_dir, kcfg_explore, spec_file, spec_module, claim_id)
+
+        # When
+        prover.advance_proof(max_iterations=1)
+        frontier_nodes = prover.proof.pending
+        assert prover.proof.status == ProofStatus.PENDING
+
+        assert len(frontier_nodes)
```

Comment:
```suggestion
        assert len(frontier_nodes) == 2
```

Perhaps more specific check?

Reply:
I copied this from `test_apr_proof_unrefute_node` above, so maybe I should make the change there as well?